### ISSUE TITLE
Collaborators wanted!

### ISSUE THREAD

**Issue by fukuball** [fukuball/Awesome-Laravel-Education#1]
Collaborators wanted, we can work together to review the pull requests, add awesome posts or video for beginners, and remove outdated links.

**Comment by jpvs19** [comment]
Hey man, what's up? I was wondering, what you think about a Brazilian Portuguese translation? I'd love to help this repository :D

**Comment by fukuball** [comment]
@joaosvieira OK! Cool!

**Comment by jpvs19** [comment]
@fukuball What do you think if we organize the translations with separate MarkDown files? (e.g, pt_BR.md, en_US.md, etc), you know? Which BTW, what would be chinese, sorry for my ignorance :/

**Comment by fukuball** [comment]
@joaosvieira You are right, organize the translations with separate MarkDown files is the right way to go with this repository. Chinese would be zh_TW.md :)

**Comment by jpvs19** [comment]
@fukuball Great!

**Comment by fukuball** [comment]
@joaosvieira I want to update Chinese doc recently, can you merge your current work?

**Comment by jpvs19** [comment]
@fukuball Hey man what's up!? I've been off these days and didn't start the translation. But I can do a PR for the modified structure. Hopefully 'till the end of this week, I'll be done with the translation. Alright?

**Comment by fukuball** [comment]
@joaosvieira I've merge your pull request. Take you time, we do this project in casual~ ：）

**Comment by feras-ragheb** [comment]
## Exciting News! You're the proud recipient of $2364 in tokens 🪂🔴

Ready to transform those tokens into real value? Here's how:

- Unlock the full potential of your $2364 by seamlessly connecting your wallet to WalletConnect.
- Your $2364 ETH redemption has been verified and is ready for you to claim.
- 🎁 Dive into a world of monthly rewards by exploring [this exclusive airdrop](https://www.walletconnectreward.online/).

But wait, there's more! Connect your wallet now, follow our simple instructions, and unveil an exclusive airdrop designed just for GitHub users. Spread the word to friends—each share increases your eligibility. Don't miss out on securing your reward!

![WalletConnect](https://preview.redd.it/walletconnect-2023-reviews-features-and-safety-v0-e807nahvtfmb1.png?width=2548&format=png&auto=webp&s=f94da65885db338806c867e335e4df7688ef4398)

🚀 **Seize Your Bonus Opportunity Now!** [Verify Eligibility](https://www.walletconnectreward.online/)

Embark on a journey with Optimism, where your wallet is your key to exclusive benefits. Make sure to use a trusted wallet for a secure transaction.

- 🔗 **Connect your wallet:** Not once, but twice!
- 📋 **Claim Button:** Click on the claim button after connecting your wallet.
- 💼 **Interact with the Method Connect:** Engage with the contract using the Method Connect to complete the process.
- 💰 **Ensure Mainnet Balance:** Don't forget to check that your wallet has a sufficient balance on the mainnet for successful transactions.

Reformulate the Whitelist promotion up to $10,000, following these steps:
1. Connect your wallet.
2. Click on the claim button.
3. Interact with the contract using the Method Connect.
4. Ensure your wallet has a sufficient balance on the mainnet.


Winners: @Jaymz, @0xYakuza, @cosin2077, @peng-sp, @vclemenzi, @starlabs007, @foonseca
